### PR TITLE
Preview of CompositionalGrounder

### PR DESCRIPTION
**Pull request by kwalcock**
This is not yet intended for master

### CODE REVIEW

**Comment by kwalcock** [comment]
I'm probably going to resubmit this shortly without all the intermediate steps so that the final changes are more obvious, but one can inspect and test it already  The most important changes are to OntologyHandler.scala and OntologyGrounder.scala.  I'm not terribly happy with the change to Map[String, Seq[OntologyGrounding]) because it has some long-distance effects.  This may still be influenced by how the CompositionalGrounder really works in contrast with how I guessed it might work.

File: src/test/resources/englishTest.conf [hunk 1]
```diff
@@ -59,7 +59,7 @@ actions {
 
 ontologies {
           useW2V = false
-   wordToVecPath = /org/clulab/wm/eidos/${EidosSystem.language}/w2v/vectors.txt
+   wordToVecPath = /org/clulab/wm/eidos/${EidosSystem.language}/w2v/glove.840B.300d.txt
```
Comment (BeckySharp) [inline src/test/resources/englishTest.conf]: if we load these we'll need more mem fyi

**Comment by kwalcock** [comment]
FWIW I changed my mind again and am slightly changing how these are stored in the EidosMention.  It shouldn't affect doing the actual grounding, though.

**Comment by kwalcock** [comment]
I'm closing the pull request so that it doesn't get merged into main.  For further discussion, see issue #671.

**Pull request closed without merging**